### PR TITLE
Made spaceship parts units instead of buildings

### PR DESCRIPTION
**Pull request by xlenstra**
Cherry-picked from #6161 so at least this can already be used and we don't have to wait until I actually have time to finish that PR.
Existing progress towards scientific victory _should_ not be lost, but I haven't tested this.

### CODE REVIEW

File: core/src/com/unciv/models/ruleset/unique/UniqueType.kt [hunk 1]
```diff
@@ -451,7 +451,8 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     CannotBeBarbarian("Never appears as a Barbarian unit", UniqueTarget.Unit, flags = listOf(UniqueFlag.HiddenToUsers)),
     
     ReligiousUnit("Religious Unit", UniqueTarget.Unit),
-    SpaceshipPart("Spaceship part", UniqueTarget.Building, UniqueTarget.Unit),
+    SpaceshipPart("Spaceship part", UniqueTarget.Unit),
```
Comment (yairm210) [inline core/src/com/unciv/models/ruleset/unique/UniqueType.kt:454]: By removing the "Building" target you make all existing mods throw errors

File: android/assets/jsons/translations/template.properties [hunk 2]
```diff
@@ -726,6 +726,8 @@ Sleep until healed =
 Moving = 
 Set up = 
 Paradrop = 
+Add in capital =
```
Comment (yairm210) [inline android/assets/jsons/translations/template.properties:729]: I think the 'every action is translated' test is causing us to have fake translations... this text would never actually be shown, if I understand correctly

File: core/src/com/unciv/models/ruleset/Building.kt [hunk 3]
```diff
@@ -475,6 +475,7 @@ class Building : RulesetStatsObject(), INonPerpetualConstruction {
                     )
                         rejectionReasons.add(RejectionReason.ShouldNotBeDisplayed)
 
+                // Shouldn't this be "Enables nuclear weapon_s_"?
```
Comment (yairm210) [inline core/src/com/unciv/models/ruleset/Building.kt:478]: I personally want this to be removed entirely and replaced by "Only available \<when [Apollo Program] has been completed]>" or somesuch

Comment (yairm210) [reply core/src/com/unciv/models/ruleset/Building.kt:478]: Currently we have the nuclear UNITS saying they need the building, and the BUILDING saying it enables the units, with no direct connection, which is suboptimal

**Review by yairm210 — CHANGES_REQUESTED** [review]
Please return the "Spaceship part" unique to units to not break existing mods

File: android/assets/jsons/translations/template.properties (hunk 2 above)
Comment (xlenstra) [reply android/assets/jsons/translations/template.properties:729]: Correct, I think this also happened before for the great engineer bonus iirc, where `Hurry Construction` is in the `template.properties` file, while only ever `Hurry Construction (+[amount]⚙)` is actually shown to the user

File: core/src/com/unciv/models/ruleset/Building.kt (hunk 3 above)
Comment (xlenstra) [reply core/src/com/unciv/models/ruleset/Building.kt:478]: Agree, though for user clarity having it both ways might be benficial

**Pull request merged**